src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -1011,14 +1012,23 @@ def get_batches_from_playbooks_and_args(
 
 
 def make_batch_file(
-    batch_file, tests_dir, ansible_args, make_batch_file_order
+    batch_file, tests_dir, ansible_args, image, make_batch_file_order
 ):
     """Create a batch file from the tests_*.yml."""
     if tests_dir is None:
         tests_dir = "tests"
     tests = glob.glob(tests_dir + "/tests_*.yml")
-    if make_batch_file_order == "alphanum":
+    if make_batch_file_order == "ascending":
         tests = sorted(tests)
+    elif make_batch_file_order == "descending":
+        tests = sorted(tests, reverse=True)
+    elif make_batch_file_order == "imagehash":
+        sha = hashlib.sha1(
+            image["name"].encode(), usedforsecurity=False
+        ).digest()
+        tests = sorted(tests, reverse=bool(sha[0] & 1))
+    elif make_batch_file_order != "natural":
+        raise ValueError("unknwon order: %s" % make_batch_file_order)
```

Comment:
```suggestion
        raise ValueError("unknown order: %s" % make_batch_file_order)
```

Reply:
:see_no_evil: thnaks for soptting my tpyo!